### PR TITLE
Updated docker cheat sheet

### PR DESCRIPTION
**Pull request by mohan08p**
#Networks and Volumes related keys added. Total 12 keys.

https://duck.co/ia/view/docker_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/docker.json [hunk 1]
```diff
@@ -39,6 +41,14 @@
                 "val": "restarts a container"
             },
             {
+                "key": "docker pause",
+                "val": "pauses a running container freezing it in place"
+            },
+            {
+                "key": "docker unpause",
+                "val": "un-suspend all processes in a container"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/docker.json]: Use `resume` rather than `un-suspend`.

**Comment by GuiltyDolphin** [comment]
@mohan08p Can you change the link in the description to point to https://duck.co/ia/view/docker_cheat_sheet

It needs to point directly at the IA page, not the ideas.

**Comment by mohan08p** [comment]
thanks @GuiltyDolphin

**Comment by moollaza** [comment]
@mohan08p it looks like this one has some conflicts, could you fix them please?

**Comment by preemeijer** [comment]
Hello @mohan08p thank you for the additions for the cheat sheet.

The only thing that I'm not agreeing with is the new description. I do thing that the original description is better to describe what docker is.

@moollaza is it necessary to start the explanation with a capital?

Cheers, Paul

**Comment by mohan08p** [comment]
@preemeijer : yeah my description was more general and yours more specific. Sorry for the inconvenience. Reverted it back. Thanks . :+1:

**Comment by preemeijer** [comment]
:+1:

**Comment by jdorweiler** [comment]
:+1:

**Pull request merged**